terraform/data/bq_views/events/pull_request_events.sql

```diff
@@ -0,0 +1,70 @@
+
+SELECT
+  received,
+  event,
+  JSON_VALUE(json_payload, "$.action") action,
+  JSON_VALUE(json_payload, "$.organization.login") ORGANIZATION,
+  INT64(JSON_QUERY(json_payload, "$.organization.id")) organization_id,
+  JSON_VALUE(json_payload, "$.repository.full_name") repository_full_name,
+  INT64(JSON_QUERY(json_payload, "$.repository.id")) repository_id,
+  JSON_VALUE(json_payload, "$.repository.name") repository,
+  JSON_VALUE(json_payload, "$.repository.visibility") repository_visibility,
+  JSON_VALUE(json_payload, "$.sender.login") sender,
+  INT64(JSON_QUERY(json_payload, "$.sender.id")) sender_id,
+  JSON_VALUE(json_payload, "$.pull_request.active_lock_reason") active_lock_reason,
+  INT64(JSON_QUERY(json_payload, "$.pull_request.additions")) additions,
+  JSON_VALUE(json_payload, "$.pull_request.base.ref") base_ref,
+  INT64(JSON_QUERY(json_payload, "$.pull_request.changed_files")) changed_files,
+  TIMESTAMP(JSON_VALUE(json_payload, "$.pull_request.closed_at")) closed_at,
+  INT64(JSON_QUERY(json_payload, "$.pull_request.comments")) comments,
+  INT64(JSON_QUERY(json_payload, "$.pull_request.commits")) commits,
+  TIMESTAMP(JSON_VALUE(json_payload, "$.pull_request.created_at")) created_at,
+  INT64(JSON_QUERY(json_payload, "$.pull_request.deletions")) deletions,
+  BOOL(JSON_QUERY(json_payload, "$.pull_request.draft")) draft,
+  JSON_VALUE(json_payload, "$.pull_request.head.ref") head_ref,
+  JSON_VALUE(json_payload, "$.pull_request.html_url") html_url,
+  INT64(JSON_QUERY(json_payload, "$.pull_request.id")) id,
+  BOOL(JSON_QUERY(json_payload, "$.pull_request.locked")) locked,
+  BOOL(JSON_QUERY(json_payload, "$.pull_request.maintainer_can_modify")) maintainer_can_modify,
+  JSON_VALUE(json_payload, "$.pull_request.merge_commit_sha") merge_commit_sha,
+  JSON_VALUE(json_payload, "$.pull_request.mergeable_state") mergeable_state,
+  BOOL(JSON_QUERY(json_payload, "$.pull_request.merged")) merged,
+  TIMESTAMP(JSON_VALUE(json_payload, "$.pull_request.merged_at")) merged_at,
+  JSON_VALUE(json_payload, "$.pull_request.merged_by.login") merged_by,
+  INT64(JSON_QUERY(json_payload, "$.pull_request.number")) number,
+  INT64(JSON_QUERY(json_payload, "$.pull_request.review_comments")) review_comments,
+  JSON_VALUE(json_payload, "$.pull_request.state") state,
+  JSON_VALUE(json_payload, "$.pull_request.title") title,
+  TIMESTAMP(JSON_VALUE(json_payload, "$.pull_request.updated_at")) updated_at,
+  INT64(JSON_QUERY(json_payload, "$.pull_request.user.id")) author_id,
+  JSON_VALUE(json_payload, "$.pull_request.user.login") author,
+  TIMESTAMP_DIFF( TIMESTAMP(JSON_VALUE(json_payload, "$.pull_request.closed_at")), TIMESTAMP(JSON_VALUE(json_payload, "$.pull_request.created_at")), SECOND) open_duration_seconds
+FROM (
+  SELECT
+    received,
+    event,
+    PARSE_JSON(payload) AS json_payload
+  FROM
+    `${dataset_id}.${table_id}`)
+WHERE
+  event = "pull_request";
+  
```

Comment:
nit: newline at end of file

Reply:
do we not want newlines at the end of the file?

Reply:
There should be, github was flagging it as missing. 🤷🏻‍♂️